WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Posts/PostsCardViewModel.swift

```diff
@@ -304,7 +304,7 @@ extension PostsCardViewModel: NSFetchedResultsControllerDelegate {
         }
         snapshot.reloadItems(reloadIdentifiers)
 
-        let shouldAnimate = viewController?.tableView.numberOfSections != 0
+        let shouldAnimate = viewController?.tableView.numberOfRows(inSection: 0) != 0
```

Comment:
I'm not sure if that case is possible or not, but if there are zero sections that line will also crash.
If that's a possible case then we can maybe change it to this:
`viewController?.tableView.numberOfSections != 0 && viewController?.tableView.numberOfRows(inSection: 0) != 0`

Reply:
Good catch! Fixed in a2b5c7ff64b0d980ae51358eeda1aff10b5b1258